catalog/dags/data_refresh/create_filtered_index_dag.py

```diff
@@ -0,0 +1,224 @@
+"""
+# Create filtered index DAG factory
+
+This module creates the filtered index creation DAGs for each media type
+using a factory function.
+
+Filtered index creation is handled by the ingestion server. The DAGs generated
+by the ``build_create_filtered_index_dag`` function in this module are
```

Comment:
Note for this docstring, this gets rendered as markdown so I believe only one set of backticks should be necessary (same for the rest of this section):
```suggestion
by the `build_create_filtered_index_dag` function in this module are
```

Reply:
Gotcha. It's hard to remember what docstrings are regular markdown and which are RST. Two backticks get rendered fine in Markdown (it's part of the CommonMark spec, even), so for consistency with all other Python docstrings I think I prefer to just use two backticks, rather than needing to think about where I do and don't need to use them inside a Python module.

Reply:
You're right it does! I didn't realize that and I thought it'd cause problems, feel free to disregard (and TIL)